docs/reference/setup/advanced-configuration.asciidoc

```diff
@@ -117,6 +117,13 @@ threshold, check the {es} log for an entry like this:
 ----
 heap size [1.9gb], compressed ordinary object pointers [true]
 ----
++
+Or check the <<cluster-nodes-info,nodes info API's>> `nodes.{name}.jvm.using_compressed_ordinary_object_pointers` value:
```

Comment:
I wonder if you can say `jvm.using_compressed_ordinary_object_pointers` under the specific node id. Curly braces have special meaning I think.

Reply:
```suggestion
Or check the `jvm.using_compressed_ordinary_object_pointers` value for the nodes using the <<cluster-nodes-info,nodes info API>>:
```